config/manager/manager.yaml

```diff
@@ -30,7 +30,7 @@ spec:
             capabilities:
               drop: ["ALL"]
           command:
-            - /manager
+            - /node-feature-discovery-operator
```

Comment:
Unrelated

Reply:
this is fixed on #53 , had to use it here to.
after merging #53 this line won't be needed